### ISSUE TITLE
Feature Request: improve Git's diff. first steps: tab size, ignore whitespace

### ISSUE THREAD

**Issue by omnipresense** [softvar/enhanced-github#47]
Sometimes I want to use github.com's file diffing ability.  
Even though it's a side-by-side view, they set the default tab size to 8.  
I found an open source chrome app that lets me adjust that
chrome.google.com/webstore/detail/github-custom-tab-size
Upshot:  ?ts=4 changes the rendering to 4

I thought 'Someone must be working on a chrome app to help with github.com use."
I see you are.  Cool.  Hope you add this feature.  
Being able to do other magic would be awesome,
Ignore whitespace in the diff is similar
?ws=1

Maybe you'll find more inspiration at github.com/tiimgreen/github-cheat-sheet

**Comment by jimwhitfield** [comment]
Yeah, what omnipresense said!  (Sheepish grin for having posted as the company.)
